lib/graphql/types.ts

```diff
@@ -1,22 +1,12 @@
 import {
+  AccountTransactionsArgs,
+  AccountTransfersArgs,
   CardType,
   Mutation,
   Query,
-  TransfersConnectionFilter,
-  TransferType,
 } from "./schema";
 
-export type FetchOptions = {
-  first?: number;
-  last?: number;
-  before?: string | null;
-  after?: string | null;
-};
-
-export type TransferFetchOptions = {
-  type: TransferType;
-  where?: TransfersConnectionFilter;
-} & FetchOptions;
+export type FetchOptions = AccountTransactionsArgs | AccountTransfersArgs;
```

Comment:
what do you think of this? 🤔 it's not strictly needed but we were once again creating some custom types that we can just get from the Schema

Reply:
Not a fan of this change (`FetchOptions`):
1. We will need to adjust this for every new collection.
2. It allows using `transaction.fetchAll` with `args` of type `AccountTransfersArgs`.

I do like that we can get rid of `TransferFetchOptions`.

Reply:
Makes total sense, I reverted that part of my changes: 0032192b41d45d1f6fd152fc7e70fa7bc1768709